xwiki-platform-core/xwiki-platform-user/xwiki-platform-user-api/src/main/java/org/xwiki/user/UserException.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.user;
+
+import org.xwiki.stability.Unstable;
+
+/**
+ * Generic exception for the user api module.
+ *
+ * @version $Id$
+ * @since 14.6RC1
+ * @since 14.4.3
+ * @since 13.10.8
+ */
+@Unstable
+public class UserException extends Exception
+{
+    private static final long serialVersionUID = 3091220482950706563L;
```

Comment:
We generally set `1` (not that it matter that much, since there is almost no chance that this exception will ever be serialized).

Reply:
I don't think it's a written rule, I was just under the impression that it's much more common in our code base, but I may be wrong.